.github/workflows/ci-release.yaml

```
@@ -0,0 +1,60 @@
+name: Release CI
+on:
+  push:
+    tags:
+      - 'release/v?[0-9]*'
+
+env:
+  RELEASE_TAG: ${{ github.ref_name }}
+
+jobs:
+  publish:
+    name: Publish to Packagist
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          path: main
+      - name: Set up environment variables
+        run: ./.github/scripts/publish_env.sh >> $GITHUB_ENV
+        working-directory: ./main
+        shell: bash
+      - name: Install PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.3"
+          tools: composer:v2
+      - name: Pack API client
+        run: ./.github/scripts/pack.sh
+        working-directory: ./main
+        shell: bash
+      - name: Create tag for Packagist (Dry run)
+        if: ${{ env.DRY_RUN == '1' }}
+        working-directory: ./main
+        run: echo "git tag v${{ env.VERSION }} && git push"
```

I guess for completeness this should match the invert:

```suggestion
        run: echo "git tag v${{ env.VERSION }} && git push origin v${{ env.VERSION }}"
```

---

good point.  I'll make that change.